apis/certificate/certificate.yaml

```diff
@@ -15,13 +17,15 @@ paths:
         This API is associated with digitally certificate generation.
            - The **_ /generate_** endpoint executes the "Certificate" request based on parameters provided as metadata in the request body to be replaced by appropriate Id
```

Comment:
Add the backend route in the description

Reply:
done